Index: src/sample/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/Main.java b/src/sample/Main.java
--- a/src/sample/Main.java	(revision 769efc282545f545d182c20bcaad82429c63df63)
+++ b/src/sample/Main.java	(date 1618924756843)
@@ -6,20 +6,22 @@
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
-public class Main /*extends Application*/ {
+public class Main extends Application {
 
-   /* @Override
+    @Override
     public void start(Stage primaryStage) throws Exception{
-        Parent root = FXMLLoader.load(getClass().getResource("sample.fxml"));
+        Parent root = FXMLLoader.load(getClass().getResource("view/sample.fxml"));
+
+
+        Scene scene = new Scene(root, 600, 600);
+
         primaryStage.setTitle("Hello World");
-        primaryStage.setScene(new Scene(root, 300, 275));
+        primaryStage.setScene(scene);
         primaryStage.show();
     }
-*/
+
 
     public static void main(String[] args) {
-        //launch(args);
-        Calculations.calc9();
-        Calculations.ConsoleVisual_basicCOCOMO();
+        launch(args);
     }
 }
Index: out/production/COCOMO/sample/sample.fxml
===================================================================
diff --git a/out/production/COCOMO/sample/sample.fxml b/out/production/COCOMO/sample/sample.fxml
deleted file mode 100644
--- a/out/production/COCOMO/sample/sample.fxml	(revision 769efc282545f545d182c20bcaad82429c63df63)
+++ /dev/null	(revision 769efc282545f545d182c20bcaad82429c63df63)
@@ -1,8 +0,0 @@
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.layout.GridPane?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<GridPane fx:controller="sample.Controller"
-          xmlns:fx="http://javafx.com/fxml" alignment="center" hgap="10" vgap="10">
-</GridPane>
\ No newline at end of file
Index: src/sample/Controller.java
===================================================================
diff --git a/src/sample/Controller.java b/src/sample/Controller.java
deleted file mode 100644
--- a/src/sample/Controller.java	(revision 769efc282545f545d182c20bcaad82429c63df63)
+++ /dev/null	(revision 769efc282545f545d182c20bcaad82429c63df63)
@@ -1,4 +0,0 @@
-package sample;
-
-public class Controller {
-}
Index: src/sample/sample.fxml
===================================================================
diff --git a/src/sample/sample.fxml b/src/sample/sample.fxml
deleted file mode 100644
--- a/src/sample/sample.fxml	(revision 769efc282545f545d182c20bcaad82429c63df63)
+++ /dev/null	(revision 769efc282545f545d182c20bcaad82429c63df63)
@@ -1,8 +0,0 @@
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.layout.GridPane?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.Label?>
-<GridPane fx:controller="sample.Controller"
-          xmlns:fx="http://javafx.com/fxml" alignment="center" hgap="10" vgap="10">
-</GridPane>
\ No newline at end of file
Index: src/sample/controllers/NeTrojSuka.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/controllers/NeTrojSuka.java b/src/sample/controllers/NeTrojSuka.java
new file mode 100644
--- /dev/null	(date 1618924081820)
+++ b/src/sample/controllers/NeTrojSuka.java	(date 1618924081820)
@@ -0,0 +1,19 @@
+package sample.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+
+public class NeTrojSuka {
+
+    @FXML
+    private ResourceBundle resources;
+
+    @FXML
+    private URL location;
+
+    @FXML
+    void initialize() {
+
+    }
+}
Index: .idea/workspace.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/workspace.xml b/.idea/workspace.xml
--- a/.idea/workspace.xml	(revision 769efc282545f545d182c20bcaad82429c63df63)
+++ b/.idea/workspace.xml	(date 1618926902069)
@@ -2,7 +2,17 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="cdb932d9-7e58-4d94-93a9-e7b232ef17da" name="Default" comment="">
-      <change beforePath="$PROJECT_DIR$/out/production/COCOMO/sample/Calculations.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/COCOMO/sample/Calculations.class" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/sample/controllers/Controller1.java" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/sample/controllers/NeTrojSuka.java" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/sample/view/ne_Troj_Suka.fxml" afterDir="false" />
+      <change afterPath="$PROJECT_DIR$/src/sample/view/sample.fxml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/COCOMO/sample/Controller.class" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/COCOMO/sample/Main.class" beforeDir="false" afterPath="$PROJECT_DIR$/out/production/COCOMO/sample/Main.class" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/out/production/COCOMO/sample/sample.fxml" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/sample/Controller.java" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/sample/Main.java" beforeDir="false" afterPath="$PROJECT_DIR$/src/sample/Main.java" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/sample/sample.fxml" beforeDir="false" />
     </list>
     <option name="SHOW_DIALOG" value="false" />
     <option name="HIGHLIGHT_CONFLICTS" value="true" />
@@ -56,6 +66,7 @@
   <component name="FileTemplateManagerImpl">
     <option name="RECENT_TEMPLATES">
       <list>
+        <option value="FxmlFile" />
         <option value="Class" />
       </list>
     </option>
@@ -150,8 +161,14 @@
     <property name="Repository.Attach.Sources" value="false" />
     <property name="WebServerToolWindowFactoryState" value="false" />
     <property name="dynamic.classpath" value="false" />
-    <property name="last_opened_file_path" value="$PROJECT_DIR$/../../../../Program Files/SceneBuilderJava/SceneBuilder.exe" />
+    <property name="last_opened_file_path" value="$PROJECT_DIR$" />
     <property name="recentsLimit" value="5" />
+  </component>
+  <component name="RecentsManager">
+    <key name="MoveFile.RECENT_KEYS">
+      <recent name="D:\labs_git\COCOMO\src\sample\view" />
+      <recent name="D:\labs_git\COCOMO\src\sample" />
+    </key>
   </component>
   <component name="RunManager">
     <configuration default="true" type="Applet" factoryName="Applet">
@@ -182,7 +199,7 @@
     <configuration name="Main" type="Application" factoryName="Application">
       <option name="MAIN_CLASS_NAME" value="sample.Main" />
       <module name="COCOMO" />
-      <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
+      <option name="VM_PARAMETERS" value="--module-path ${PATH_TO_FX} --add-modules javafx.controls,javafx.fxml" />
       <extension name="coverage">
         <pattern>
           <option name="PATTERN" value="sample.*" />
@@ -293,6 +310,18 @@
   <component name="VcsManagerConfiguration">
     <option name="OFFER_MOVE_TO_ANOTHER_CHANGELIST_ON_PARTIAL_COMMIT" value="true" />
     <option name="INCLUDE_TEXT_INTO_SHELF" value="false" />
+  </component>
+  <component name="XDebuggerManager">
+    <breakpoint-manager>
+      <breakpoints>
+        <line-breakpoint enabled="true" type="java-field">
+          <url>file://$PROJECT_DIR$/src/sample/controllers/Controller1.java</url>
+          <line>29</line>
+          <properties field="Rbtn4" class="sample.controllers.Controller1" />
+          <option name="timeStamp" value="1" />
+        </line-breakpoint>
+      </breakpoints>
+    </breakpoint-manager>
   </component>
   <component name="editorHistoryManager">
     <entry file="file://$PROJECT_DIR$/src/sample/Main.java">
Index: src/sample/view/sample.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/view/sample.fxml b/src/sample/view/sample.fxml
new file mode 100644
--- /dev/null	(date 1618928169237)
+++ b/src/sample/view/sample.fxml	(date 1618928169237)
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.MenuButton?>
+<?import javafx.scene.control.MenuItem?>
+<?import javafx.scene.control.RadioButton?>
+<?import javafx.scene.control.ToggleGroup?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.text.Font?>
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/16" xmlns:fx="http://javafx.com/fxml/1" fx:controller="sample.controllers.Controller1">
+   <children>
+      <Button fx:id="Btn1" layoutX="6.0" layoutY="368.0" mnemonicParsing="false" text="Button" />
+      <Button fx:id="Btn2" layoutX="533.0" layoutY="368.0" mnemonicParsing="false" text="Button" />
+      <Label fx:id="KekLbl1" layoutX="91.0" layoutY="152.0" prefHeight="124.0" prefWidth="462.0" text=" " textAlignment="CENTER">
+         <font>
+            <Font size="85.0" />
+         </font>
+      </Label>
+      <MenuButton fx:id="MenuTeam" layoutX="170.0" layoutY="18.0" mnemonicParsing="false" text="выберите">
+        <items>
+          <MenuItem fx:id="option1" mnemonicParsing="false" text="Распространенный" />
+          <MenuItem fx:id="option2" mnemonicParsing="false" text="Полунезависимый" />
+            <MenuItem fx:id="option3" mnemonicParsing="false" text="Встроенный" />
+        </items>
+      </MenuButton>
+      <Label fx:id="Lbl11" layoutX="14.0" layoutY="14.0" prefHeight="25.0" prefWidth="160.0" text="Тип команды :" textAlignment="CENTER">
+         <font>
+            <Font size="22.0" />
+         </font>
+      </Label>
+      <GridPane gridLinesVisible="true" layoutY="305.0" prefHeight="32.0" prefWidth="600.0">
+        <columnConstraints>
+          <ColumnConstraints hgrow="SOMETIMES" maxWidth="116.0" minWidth="10.0" prefWidth="116.0" />
+            <ColumnConstraints hgrow="SOMETIMES" maxWidth="80.0" minWidth="10.0" prefWidth="65.0" />
+            <ColumnConstraints hgrow="SOMETIMES" maxWidth="85.0" minWidth="10.0" prefWidth="65.0" />
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="65.0" />
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="65.0" />
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="65.0" />
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="65.0" />
+        </columnConstraints>
+        <rowConstraints>
+          <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+        </rowConstraints>
+         <children>
+            <Label fx:id="indicator1" prefHeight="45.0" prefWidth="108.0" text="1" GridPane.columnIndex="1">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <Label fx:id="indicator2" prefHeight="87.0" prefWidth="81.0" text="1" GridPane.columnIndex="2">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <Label fx:id="indicator3" prefHeight="89.0" prefWidth="295.0" text="1" GridPane.columnIndex="3">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <Label fx:id="indicator4" prefHeight="145.0" prefWidth="367.0" text="1" GridPane.columnIndex="4">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <Label fx:id="indicator6" prefHeight="86.0" prefWidth="175.0" text="1" GridPane.columnIndex="5">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <Label fx:id="indicator6" prefHeight="143.0" prefWidth="205.0" text="1" textAlignment="CENTER" GridPane.columnIndex="6">
+               <font>
+                  <Font size="25.0" />
+               </font>
+            </Label>
+            <MenuButton fx:id="MenuTeam1" mnemonicParsing="false" text="выберите">
+               <items>
+                  <MenuItem fx:id="option11" mnemonicParsing="false" text="Распространенный" />
+                  <MenuItem fx:id="option21" mnemonicParsing="false" text="Полунезависимый" />
+                  <MenuItem fx:id="option31" mnemonicParsing="false" text="Встроенный" />
+               </items>
+            </MenuButton>
+         </children>
+      </GridPane>
+      <RadioButton fx:id="Rbtn1" layoutX="147.0" layoutY="285.0" mnemonicParsing="false">
+         <toggleGroup>
+            <ToggleGroup fx:id="group1" />
+         </toggleGroup>
+      </RadioButton>
+      <RadioButton fx:id="Rbtn2" layoutX="230.0" layoutY="285.0" mnemonicParsing="false" toggleGroup="$group1" />
+      <RadioButton fx:id="Rbtn3" layoutX="314.0" layoutY="285.0" mnemonicParsing="false" toggleGroup="$group1" />
+      <RadioButton fx:id="Rbtn4" layoutX="386.0" layoutY="285.0" mnemonicParsing="false" toggleGroup="$group1" />
+      <RadioButton fx:id="Rbtn5" layoutX="551.0" layoutY="285.0" mnemonicParsing="false" toggleGroup="$group1" />
+      <RadioButton fx:id="Rbtn6" layoutX="468.0" layoutY="285.0" mnemonicParsing="false" prefHeight="0.0" prefWidth="16.0" toggleGroup="$group1" />
+   </children>
+</AnchorPane>
Index: src/sample/view/ne_Troj_Suka.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/view/ne_Troj_Suka.fxml b/src/sample/view/ne_Troj_Suka.fxml
new file mode 100644
--- /dev/null	(date 1618923945461)
+++ b/src/sample/view/ne_Troj_Suka.fxml	(date 1618923945461)
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ScrollPane?>
+<?import javafx.scene.layout.AnchorPane?>
+
+<ScrollPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="525.0" prefWidth="600.0" xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/16">
+  <content>
+    <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="200.0" prefWidth="200.0" />
+  </content>
+</ScrollPane>
Index: src/sample/controllers/Controller1.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/controllers/Controller1.java b/src/sample/controllers/Controller1.java
new file mode 100644
--- /dev/null	(date 1618927058193)
+++ b/src/sample/controllers/Controller1.java	(date 1618927058193)
@@ -0,0 +1,88 @@
+package sample.controllers;
+
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.scene.control.*;
+import  javafx.scene.control.MenuButton;
+
+public class Controller1 {
+    @FXML
+private ResourceBundle resources;
+
+    @FXML
+    private URL location;
+
+    @FXML
+    private RadioButton Rbtn1;
+
+    @FXML
+    private ToggleGroup group1;
+
+    @FXML
+    private RadioButton Rbtn2;
+
+    @FXML
+    private RadioButton Rbtn3;
+
+    @FXML
+    private RadioButton Rbtn4;
+
+    @FXML
+    private RadioButton Rbtn5;
+
+    @FXML
+    private RadioButton Rbtn6;
+
+    @FXML
+    private Button Btn1;
+
+    @FXML
+    private Button Btn2;
+    @FXML
+    private MenuItem option1;
+
+    @FXML
+    private MenuItem option2;
+
+    @FXML
+    private MenuItem option3;
+
+    @FXML
+    private Label Lbl1;
+    @FXML
+    private MenuButton MenuTeam;
+
+    @FXML
+    void initialize() {
+       option1.setOnAction(Event -> {
+           MenuTeam.setText("Распространенный");
+       });
+        option2.setOnAction(Event -> {
+            MenuTeam.setText("Полунезависимый");
+        });
+        option3.setOnAction(Event -> {
+            MenuTeam.setText("Встроенный");
+        });
+
+
+
+
+
+
+
+        Rbtn1.setOnAction(event ->{Lbl1.setText("Suka");});
+        Rbtn2.setOnAction(event ->{Lbl1.setText("blyat");});
+        Rbtn3.setOnAction(event ->{Lbl1.setText("pidr");});
+        Rbtn4.setOnAction(event ->{Lbl1.setText("ebanniy");});
+        Rbtn5.setOnAction(event ->{Lbl1.setText("govno");});
+        Rbtn6.setOnAction(event ->{Lbl1.setText("jopa");});
+
+
+
+
+    }
+}
+
+
